libs/vkd3d/command.c

```diff
@@ -12006,6 +12117,12 @@ static void d3d12_command_queue_execute(struct d3d12_command_queue *command_queu
     submit_desc[num_submits - 1].commandBufferCount = count;
     submit_desc[num_submits - 1].pCommandBuffers = cmd;
 
+    submission_timeline_count = ++vkd3d_queue->submission_timeline_count;
```

Comment:
Where are execute commands waiting on the queued fence waits?  I can't seem to find where we set pWaitSemaphores and so forth for the execute submission info.